src/data/constants/testData.js

```diff
@@ -0,0 +1,164 @@
+const courseOptions = {
+  data: {
+    actions: {
+      POST: {
+        type: {
+          type: 'field',
+          required: false,
+          read_only: false,
+          label: 'Type',
+          type_options: [
+            {
+              course_run_types: [
+                {
+                  uuid: 'f17e29d6-4648-4bb5-a199-97dc40f904aa',
```

Comment:
All these real UUIDs -- did you grab devstack output?

Reply:
Yeah, they are from my local options response. Is that bad? Should I dummy them out to something else?

Reply:
Naw, I just thought it was funny.